samples/bot-types-of-cards/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-bot-types-of-cards-nodejs",
+    "source": "officeDev",
+    "title": "Types of Cards",
+    "shortDescription": "Demonstrating the feature where user can send types of cards using bot.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-types-of-cards/nodejs",
+    "longDescription": [
+      "Demonstrating the feature where user can send types of cards using bot."
```

Comment:
Demonstrating the feature where user can send different types of cards using bot.

Reply:
fixed